### PR TITLE
Rename framework_framework variable

### PR DESCRIPTION
**Pull request by samuelhwilliams**
## Summary
`framework_framework` is an unnecessarily obtuse variable name and is
not overly descriptive of what it actually means. When used, this
describes the 'family' of framework being referenced. For example,
G-Cloud 10 is in the 'G-Cloud' family of frameworks. Digital Outcomes
and Specialists 2 is in the 'Digital Outcomes and Specialists' family.
Let's change all the references of `framework_framework` to
`framework_family`.

Bumps version to 38.0.0

 ## Ticket
https://trello.com/c/iqFcVKpd/14

### CODE REVIEW

**Comment by risicle** [comment]
Hey, @galund did you know that the man behind the curtain is in plain view? :house: :girl: :dog2: :tornado: :lion: :heart: :monkey: :rainbow:

File: dmutils/api_stubs.py [hunk 1]
```diff
@@ -139,7 +139,7 @@ def brief(status="draft",
             "title": "I need a thing to do a thing",
             "frameworkSlug": framework_slug,
             "frameworkName": framework_name,
-            "frameworkFramework": framework_framework,
+            "frameworkFramework": framework_family,
```
Comment (katstevens) [inline dmutils/api_stubs.py]: Our api stub for `Brief` is out of date compared with our `Brief.serialize()` method. As well as the top level `frameworkFoo` keys (which are due to be deprecated), we also have a nested `framework` dict with `family, name, slug, status`: https://github.com/alphagov/digitalmarketplace-api/blob/master/app/models/main.py#L1574.  

For bonus fun, spot my hilarious typo in that TODO comment ... 🤦‍♀️

**Comment by samuelhwilliams** [comment]
I'd like to fix/extend the stubs as part of a separate ticket and ideally just keep this as the minimum update for this story. Otherwise it would spiral :(

**Comment by katstevens** [comment]
Hmm, OK. Can we capture it as part of this existing tech debt ticket? https://trello.com/c/FYPvix7N/183-test-utils-rationalize-apiclient-test-stubs (that ticket says there are also stubs in the API client! Do they need updating too?)

**Comment by samuelhwilliams** [comment]
There are no longer stubs in the apiclient -  I moved them to utils. That ticket is probably out of date now.

**Comment by samuelhwilliams** [comment]
But I've updated the ticket to talk about updating stubs in dm-utils and I'll probably take that as one of my next tech debt tickets. @katstevens So that sorted, is this good?

**Pull request merged**